### DISCUSSION TITLE
Insights: time series does not show data with dates in future

### DISCUSSION THREAD

**Discussion by andrenarchy** [directus/directus#14712] [Feature Requests]
### Describe the Bug

The time series panel ignores data with date fields in the future and it simply shows "no data". The use case for us is to provide a forecast of when projects are scheduled to complete:
![image](https://user-images.githubusercontent.com/1874116/181481946-a3637b4b-3cf2-4291-a6d9-c6603dd45573.png)

The UI seems a bit unintuitive because the dropdown only shows "Past X" entries. Maybe it makes sense to split this into two fields (`from`/`to`) where "now" would be an option in both?

### To Reproduce

Create a time series panel with a date field that contains dates in the future.

### What version of Directus are you using?

9.14.1

### What version of Node.js are you using?

16

### What database are you using?

Postgresql 14

### What browser are you using?

Chrome

### How are you deploying Directus?

k8s

**Comment by HuldaCZ** [comment]
Hello, we where looking for similar solution to be able to select specific range on time-series, not just $NOW() - xxx. 
So I implement automatic range option based on data inserted to the series. 

You can select Automatic range option and then set filter to match specific data. 
In case there is no filter it will get all data for selected collection. 

![image](https://user-images.githubusercontent.com/49844489/185406867-a0f23bdd-c054-4f96-956a-6050ab335bbf.png)

@rijkvanzanten Can I create PR?

**Reply by rijkvanzanten** [reply]
What are the other options under date range? Is that the same field we have now? If that's the case (and it's backwards compatible with no breaking changes), then yes by all means go ahead with a PR 🙂

**Reply by HuldaCZ** [reply]
Yes it's the same filed. The other options are without changes. Backwards compatibility should be ok. 

![image](https://user-images.githubusercontent.com/49844489/185858823-0222e33e-0da0-4575-a5f5-4d686ea9ecb8.png)

**Reply by HuldaCZ** [reply]
PR: https://github.com/directus/directus/pull/15200

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
